allennlp/modules/seq2vec_encoders/bert_pooler.py

```diff
@@ -26,18 +27,25 @@ class BertPooler(Seq2VecEncoder):
         The pretrained BERT model to use. If this is a string,
         we will call ``BertModel.from_pretrained(pretrained_model)``
         and use that.
+    dropout : ``float``, optional, (default = 0.0)
+        Amount of dropout to apply after pooling
     requires_grad : ``bool``, optional, (default = True)
         If True, the weights of the pooler will be updated during training.
         Otherwise they will not.
     """
-    def __init__(self, pretrained_model: Union[str, BertModel], requires_grad: bool = True) -> None:
+    def __init__(self,
+                 pretrained_model: Union[str, BertModel],
+                 dropout: float = 0.0,
```

Comment:
Please could you change the position of this argument to be last, so that it doesn't introduce a breaking change?

Reply:
Ok, done